src/include/execution/sql/decimal_magic_numbers.h

```diff
@@ -0,0 +1,269 @@
+#pragma once
+
+#include <unordered_map>
+
+#include "storage/storage_defs.h"
```

Comment:
What does this file need `storage_defs.h` for?

Reply:
It doesn't i'll fix this

Reply:
resolved in latest PR